cfpb/sbl-frontend
# feat(poc): Point of Contact - Skeleton

closes #287 

## Changes
- style(Fieldgroup): Adhere's to the 770px max-width with box-border (includes borders, margins and padding into the max-width)
- feat(Point of Contact): Skeleton Implemented - awaiting functionality

## How to Test
- If needed, `toggleRouting()` to disable enforced routing
- Navigate to `http://localhost:8899/point-of-contact` to verify
